contracts/JBReconfigurationBufferBallot.sol

```diff
@@ -37,7 +37,13 @@ contract JBReconfigurationBufferBallot is ERC165, IJBFundingCycleBallot {
 
     unchecked {
       // If there was sufficient time between configuration and the start of the cycle, it is approved. Otherwise, it is failed.
-      return (_start - _configured < duration) ? JBBallotState.Failed : JBBallotState.Approved;
+      // If the ballot hasn't yet started, it's state is ApprovalExpected.
+      return
+        (_start - _configured < duration)
+          ? JBBallotState.Failed
+          : (block.timestamp < _start - duration)
```

Comment:
cf https://discord.com/channels/775859454780244028/786985806396784690/1173612390710001694
should include a constructor check duration < block.timestamp (or just uncheck this block)